app/Http/Controllers/MainSiteController.php

```diff
@@ -2,13 +2,26 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Requests\Main\ContactUsRequest;
+use App\Models\ContactUs;
+
 class MainSiteController extends Controller
 {
     public function index()
     {
         return view('main.home');
     }
 
+    public function contactUs(ContactUsRequest $request)
+    {
+        $contact = new ContactUs();
+        $contact->name = $request->txtname;
+        $contact->email = $request->txtemail;
+        $contact->message = $request->txtmessage;
+        $contact->save();
+        return view('main.home');
```

Comment:
Don't do it like this. just redirect to main.home route.

Reply:
Done Resolved